config/config.go

```diff
@@ -4,15 +4,27 @@
 package config
 
 import (
+	"fmt"
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"runtime"
 )
 
 const (
 	DiceDBVersion = "0.1.0"
+
+	linuxPath   = "/var/lib/dicedb"
+	darwinPath  = "~/Library/Application Support/dicedb"
+	windowsPath = "C:\\ProgramData\\dicedb"
```

Comment:
A better way to handle this is by adding OS specific files and let Go's compiler take care of binding the apt file. You can import this variable in `constants.go` file and refer that variable in the codebase.

This way, you don't need multiple variables.